core/management/commands/initialize_historical_counters.py

```diff
@@ -0,0 +1,117 @@
+from funcy import zipdict, isums, walk_values, group_by, count_by, second, compose, first
+from tqdm import tqdm
+from handy.db import queryset_iterator
+from datetime import datetime
+from django.db.models import Count
+from django.db import transaction
+from django.core.management import BaseCommand
+from core.models import HistoricalCounter
+from legacy.models import Series, Sample, Platform, PlatformProbe
+from core.models import User
+from tags.models import Tag, SerieAnnotation, SampleAnnotation
+
+MONTHS = ['Jan', 'Feb', 'Mar', 'Apr', 'May', 'Jun', 'Jul', 'Aug', 'Sep', 'Oct', 'Nov', 'Dec']
+
+def floor_attrs_date(model):
+    month, day, year = model.attrs.get('submission_date', 'Jan 1 1960').split()
+    if month == 'Jans':
+        month = 'Jan'
+    return datetime(int(year), MONTHS.index(month) + 1, 1)
+
+def floor_date(date):
+    return datetime(date.year, date.month, 1)
```

Comment:
Should use ceiling not floor - next 1st of the month. By that date those things will be in. Not by the 1st of the same month.